examples/pytorch/gat/gat.py

```diff
@@ -44,7 +47,8 @@ def __init__(self, headid, indim, hiddendim, activation, residual):
         self.residual_fc = None
         if residual:
             if indim != hiddendim:
-                self.residual_fc = nn.Linear(indim, hiddendim)
+                self.residual_fc = nn.Linear(indim, hiddendim, bias=False)
+                nn.init.xavier_normal_(self.residual_fc.weight.data, gain=1.414)
```

Comment:
Would it better to change to `gain=nn.init.calculate_gain('relu'))`? which is equivalent to `gain=1.414`